src/content.js

```diff
@@ -5,6 +5,9 @@ const buildData = require("./data");
 const fm = require("front-matter");
 const fs = require("fs").promises;
 const path = require("path");
+const MiniSearch = require("minisearch");
+
+const STOP_WORDS = ["and", "or", "to", "at", "in", "a", "the", "be", "are", "is", "as", "its", "it", "this", "these", "any"];
```

Comment:
When building the search index, we don't need to consider these terms. This slightly reduces the index size.